msbuild/Xamarin.iOS.Tasks.Core/Tasks/DetectSdkLocationsTaskBase.cs

```diff
@@ -88,6 +88,7 @@ public abstract class DetectSdkLocationsTaskBase : Task
 
 		public override bool Execute ()
 		{
+			var name = typeof (LoggingExtensions).Assembly.FullName;
```

Comment:
how would that local variable be needed ?

Reply:
Oops, artifact of debugging :)